### ISSUE TITLE
Add context menu shortcuts to keyboard shortcuts popup

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#8440]
There's some new keyboard shortcuts introduced in the right click context menus that are missing from the keyboard shortcut popup. Let's add those in and make sure shortcuts are available.
